src/Auth0.ManagementApi/Clients/IGrantsClient.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Auth0.ManagementApi.Models.Grants;
+using Auth0.ManagementApi.Paging;
+
+namespace Auth0.ManagementApi.Clients
+{
+    public interface IGrantsClient
+    {
+        /// <summary>
+        /// Retrieve all grants.
```

Comment:
Should this `grants` be capitalized or should the others (line 23 and 34) be lowercased?

Reply:
```suggestion
        /// Retrieve all Grants.
```